### DISCUSSION TITLE
windows 11

### DISCUSSION THREAD

**Discussion by SandDoller** [artisan-roaster-scope/artisan#1733] [General]
hi 
how i can install artisan with windows 11?

**Comment by roasterdave** [comment]
https://artisan-scope.org/download/

**Comment by pencoat** [comment]
I have an issue also with the 3.02 and 3.03 version, McAfee or Microsoft Antivirus see a threat inside and put the file in quarantine. I've tried to stop my antivirus while I installed the program but McAfee still found a threat after and put the .exe in quarantine.  So for now, I don't have Artisan, unless I reinstall an older version

**Reply by poundy** [reply]
3.03 isn't a full release so it's hardly worth trying to use it unless you need one of the imminent fixes. 
This is a false positive from McAfee, nothing Artisan can do to help you unfortunately. Up to you to whitelist the app until they recognise it's legitimacy.
